heralding/capabilities/http.py

```diff
@@ -71,19 +65,18 @@ def do_GET(self):
             headers_bytes = bytes(self.headers['Authorization'], 'utf-8')
             self.wfile.write(headers_bytes)
             self.wfile.write(b'not authenticated')
-        self.request.close()
 
     # Disable logging provided by BaseHTTPServer
     def log_message(self, format_, *args):
         pass
 
 
 class Http(HandlerBase):
-    HandlerClass = BeeHTTPHandler
-
-    def __init__(self, options):
-        super().__init__(options)
+    def __init__(self, options, loop):
+        super().__init__(options, loop)
         self._options = options
 
-    def execute_capability(self, address, gsocket, session):
-        self.HandlerClass(gsocket, address, None, httpsession=session, options=self._options)
\ No newline at end of file
+    async def execute_capability(self, reader, writer, session):
+        http_cap = BeeHTTPHandler(reader, writer, httpsession=session, options=self._options)
+        await http_cap.run()
```

Comment:
Does this need to be async here? What would happen if we removed this await (and the async in the def)?